codec/decode.go

```diff
@@ -125,6 +127,12 @@ func (sd *Decoder) DecodeFixedWidthInt(t interface{}) (o int, err error) {
 		if err == nil {
 			o = int(binary.LittleEndian.Uint32(buf))
 		}
+	case int64, uint64:
+		buf := make([]byte, 8)
+		_, err = sd.Reader.Read(buf)
+		if err == nil {
+			o = int(binary.LittleEndian.Uint64(buf))
```

Comment:
i don't think it should make a difference since the underlying bits stay the same, and the actual decode function returns an interface so it's up to the person decoding to cast it to the right type. i can change it to uints though if you're like

Reply:
That is true, but interfaces still have underlying types which may determine how they are interpreted

Think about our favorite switch statements:
```
switch v := myInterface.(type) {
case int:
    // Oh hey, it's an int!
case uint:
     // UINT! UINT! UINT!
```

Reply:
hmm okay, I will change to uint

Reply:
done